Gemfile.lock

```diff
@@ -943,4 +944,4 @@ RUBY VERSION
    ruby 2.7.5p203
 
 BUNDLED WITH
-   2.3.6
+   2.3.5
```

Comment:
why?

Reply:
It was the bundler job, should I quit this file?

Reply:
No, but why the version was downgrade to 2.3.5?

Reply:
Fixed